src/features/student/queries/index.ts

```diff
@@ -1 +1,2 @@
+export * from '@/features/student/queries/use-student-access-control.query'
 export * from '@/features/student/queries/use-student-summary.query'
```

Comment:
```suggestion
export * from './use-student-access-control.query'
export * from './use-student-summary.query'
```

Comme c'est un barrel file, c'est ok de faire un export en relatif

Reply:
bien vu